docs/00a-preamble/01-what-to-expect-of-an-advanced-level-module.adoc

```diff
@@ -9,6 +9,9 @@ Das Modul kann unabhängig von einer CPSA-F-Zertifizierung besucht werden.
 
 // tag::EN[]
 === What is taught in an Advanced Level module?
+
+The module can be attended independently of a CPSA-F certification.
```

Comment:
Das sollte dann ins Template und dann in alle Lehrpläne.

Reply:
added a ticket: https://github.com/isaqb-org/advanced-template/issues/161